src/json/state_machine_input.json

```diff
@@ -142,6 +142,7 @@
     },
     "result": null
   },
+  "share_portfolio_params": null,
```

Comment:
Define this as 

{
          aws_region_id: null,
          aws_account_id: null
          role_details: {
            role_name: null,
            external_id: null
          }
}